### PR TITLE
Updated esdc-dns appliance and first compute node installer to configure recursion forwarders

### PR DESCRIPTION
**Pull request by dn0**
The recursion forwarders will be configured according to DNS resolvers, which were set during installation.
Added new esdc-mon mdata: `org.erigones:recursor_forwarders`.

### CODE REVIEW

File: ansible/roles/pdns-recursor/templates/recursor.conf.j2 [hunk 1]
```diff
@@ -88,6 +88,7 @@ config-dir={{ recursor_config_dir }}
 # forward-zones	Zones for which we forward queries, comma separated domain=ip pairs
 #
 # forward-zones=
+{% if recursor_forward_zones %}forward-zones={{ recursor_forward_zones | join(',') }}{% endif %}
```
Comment (YanChii) [inline ansible/roles/pdns-recursor/templates/recursor.conf.j2:91]: I think this line is not needed.
https://doc.powerdns.com/md/recursor/settings/#forward-zones

File: ansible/templates/usb/scripts/headnode.sh.j2 [hunk 2]
```diff
@@ -191,6 +191,7 @@ function _build_vmmanifest() {
 				-e "s|@MON_IP@|${MON_IP}|" \
 				-e "s|@ROOT_AUTHORIZED_KEYS@|${ROOT_AUTHORIZED_KEYS}|" \
 				-e "s|@PGSQL_PDNS_PASSWORD@|${PGSQL_PDNS_PASSWORD}|" \
+				-e "s|@PDNS_RECURSOR_FORWARDERS@|.=${DNS_RESOLVERS/,/;}|" \
```
Comment (YanChii) [inline ansible/templates/usb/scripts/headnode.sh.j2]: We really should set this mdata conditionally. We don't want to flood (default) Google DNS servers by every recursive query in the cloud. By default, our pdns server should take care of all recursive queries by itself (opposite to asking a "smarthost"). 
Please add (in pseudo code):
```if recursors != 8.8.[48].[48]; then set metadata...```

**Review by YanChii — CHANGES_REQUESTED** [review]

File: ansible/roles/pdns-recursor/templates/recursor.conf.j2 (hunk 1 above)
Comment (dn0) [reply ansible/roles/pdns-recursor/templates/recursor.conf.j2:91]: Yep. Not needed, but does not hurt - just a feature for the pdns ansible role.

File: ansible/templates/usb/scripts/headnode.sh.j2 (hunk 2 above)
Comment (dn0) [reply ansible/templates/usb/scripts/headnode.sh.j2]: Done.

**Review by YanChii — APPROVED** [review]

**Pull request merged**